tailwind.config.ts

```diff
@@ -123,6 +123,14 @@ export default {
         },
       ],
     },
+    boxShadow: {
+      soft: "0 2px 20px 0 hsl(var(--neutral-100) / 0.04)",
+      DEFAULT: "0 4px 20px 0 hsl(var(--neutral-100) / 0.08)",
+      intense: "0 8px 30px 0 hsl(var(--neutral-100) / 0.12)",
+      strong: "0 12px 56px 0 hsl(var(--neutral-100) / 0.16)",
+      inner: "inset 0 2px 4px 0 hsl(var(--neutral-100) / 0.04)",
+      none: "none",
```

Comment:
We're diverging quite a lot from `shadcn` and `tailwind`, so importing components will be more difficult. I guess that's ok? I still don't see why we don't just stick to the standard names (`xs`, `sm`, `md`...). They are actually quite informative. Looking at this list, I'm unsure about the difference between `strong` or `intense` for example.

@dani-moreno what do you think?

Reply:
happy to get back to standard names as long as we update sync it with figma

Reply:
Yes, I completely agree. It's easier to adapt Figma to code rather than the other way around. Plus, Tailwind has a good naming convention, in my opinion.

I've updated the naming in Figma.

Reply:
Awesome! Can we do the same for the rest of customizations?

Reply:
Which ones?

Reply:
You know what? None 🤣. My bad, I thought there were others but I see this was the only exception. Good!

Reply:
fixed. The names are updated and the inset is removed